cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -366,7 +367,10 @@ func GetAvailableResources(clientBuilder controller.ControllerClientBuilder) (ma
 
 	resourceMap, err := discoveryClient.ServerResources()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get supported resources from server: %v", err)
+		utilruntime.HandleError(fmt.Errorf("unable to get all supported resources from server: %v", err))
```

Comment:
> This means that we continue, go into the if clause below and return an error. What have we won with this change?

it can return an error and a the results it was able to get.  Consider the aggregated case with one server of ten being down.